pkg/crc/preflight/preflight_daemon_task_check_windows.go

```diff
@@ -142,7 +142,7 @@ func checkIfDaemonTaskRunning() error {
 	if strings.TrimSpace(stdout) != "Running" {
 		return fmt.Errorf("expected %s task to be in 'Running' but got '%s'", constants.DaemonTaskName, stdout)
 	}
-	return nil
+	return waitForDaemonRunning()
```

Comment:
on a clean system this is always going to fail since we are calling `waitForDaemonRunning` in the `check` itself, and we'll be waiting for the retries to finish.

this `waitForDaemonRunning()` should be called from the `fixDaemonTaskRunning` where after starting the daemon, we retry and block till its able to serve